### DISCUSSION TITLE
Replacement of the FederatedCatalog with another dcat based catalog

### DISCUSSION THREAD

**Discussion by arianman** [eclipse-edc/FederatedCatalog#121] [Q&A]
May I ask what would be the easiest way to integrate another catalog with the EDC?

I've had a look at the following repos:
https://github.com/eclipse-edc/FederatedCatalog
https://github.com/eclipse-edc/Connector/tree/main/extensions/control-plane/api/management-api/catalog-api

Where would you recommend to start/look into/make the change?

Many thanks!

**Comment by jimmarino** [comment]
This is a bit of an open-ended question that is difficult to answer. It depends on what you want to achieve.

Please look at the [DSP Specification](https://docs.internationaldataspaces.org/dataspace-protocol/overview/readme) and the `AssetIndex` interface.
